[Bugs]Got errors in ArchLinux

```
[main-process] undefined: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/lvce/app/packages/shared-process/node_modules/@lvce-editor/pty-host/src/ptyHostMain.js' imported from /usr/lib/lvce/: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/lvce/app/packages/shared-process/node_modules/@lvce-editor/pty-host/src/ptyHostMain.js' imported from /usr/lib/lvce/
```
```
VError: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/lvce/app/packages/shared-process/node_modules/@lvce-editor/pty-host/src/ptyHostMain.js' imported from /usr/lib/lvce/: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/lvce/app/packages/shared-process/node_modules/@lvce-editor/pty-host/src/ptyHostMain.js' imported from /usr/lib/lvce/
    at Object.create$2 [as create] (file:///usr/lib/lvce/app/packages/main-process/dist/mainProcessMain.js:5065:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.create$1 [as IpcParent.create] (file:///usr/lib/lvce/app/packages/main-process/dist/mainProcessMain.js:5418:18)
    at async getResponse (file:///usr/lib/lvce/app/packages/main-process/dist/mainProcessMain.js:3937:109)
    at async handleJsonRpcMessage (file:///usr/lib/lvce/app/packages/main-process/dist/mainProcessMain.js:3947:24)

[main process] shared process exited with code 0
```

--- a/packages/build/src/parts/FilterSharedProcessDependencies/FilterSharedProcessDependencies.js
+++ b/packages/build/src/parts/FilterSharedProcessDependencies/FilterSharedProcessDependencies.js
@@ -12,9 +12,6 @@ export const filterDependencies = (rawDependencies, exclude = []) => {
     if (dependency.name === '@lvce-editor/extension-host') {
       return false
     }
-    if (dependency.name === '@lvce-editor/pty-host') {
-      return false
-    }
     if (dependency.name === 'prebuild-install') {
       return false
     }

